TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Helper/GamePad.java

```diff
@@ -139,6 +142,7 @@ public enum GameplayInputType {
     // Class Constructor
     public GamePad(@NonNull Gamepad gp) {
         this.inputGPad = gp;
```

Comment:
a instantiated `GamepadHaptics` class will be available from the `Gamepad` class itself for ease of use